src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline

```diff
@@ -5776,12 +5783,10 @@ require "library"
 
 # [START hopper]
 
- # Testing resource name overlap
- #
- # @param shelf {String} Test word wrapping for long lines. This is a long comment. The name of the
- # shelf to retrieve the big book from.
- # @param big_book_name {String} The name of the book.
-def sample_get_big_book(shelf, big_book_name)
+
+
+
+def sample_get_big_book shelf, big_book_name
```

Comment:
Shouldn't this have parens now?

Reply:
I thought rubocop does not like parens?

Copying from rubocop report:

```
samples/v1/speech_transcribe_multichannel.rb:24:21: C: Style/MethodDefParentheses: Use def without parentheses.
def sample_recognize(local_file_path)
                    ^^^^^^^^^^^^^^^^^
```

Or does that only apply to the case where there is only one parameters?

Reply:
Ohhhh, we _had_ them before, I didn't realize we do.

Ok, +1

I couldn't remember what ruby-style's option was of this. Nvm, LGTM :D